packages/runtime/src/runtime/menu.tsx

```diff
@@ -154,28 +173,29 @@ export const SideBarMenu: React.FC<MenuBaseProps> = ({ id, ...props }) => {
           <>
             <AntMenu.Item
               data-testid={item.id ?? item.testId}
-              icon={renderMuiIcon(
-                item.icon,
-                props.styles?.iconWidth,
-                props.styles?.iconHeight,
+              icon={getIcon(
+                selectedItem === item.page
+                  ? item.activeIcon || item.icon
+                  : item.icon,
+                selectedItem === item.page ? "activeIcon" : "icon",
```

Comment:
Can we combine this into a single function so we can clean up the logic with if/else instead of ternary?